vapi/rest/client.go

```diff
@@ -179,6 +180,11 @@ func (c *Client) Do(ctx context.Context, req *http.Request, resBody interface{})
 		}
 	}
 
+	// OperationID (see soap.Client.soapRoundTrip)
+	if id, ok := ctx.Value(types.ID{}).(string); ok {
```

Comment:
Had no idea you could use an empty var as a value key.  Makes perfect sense.  Very elegant.